src/main/java/com/st/utopia/cancellationservice/service/TicketService.java

```diff
@@ -0,0 +1,52 @@
+package com.st.utopia.cancellationservice.service;
+
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import com.st.utopia.cancellationservice.dao.TicketDao;
+import com.st.utopia.cancellationservice.model.Ticket;
+
+
+
+@Service
+public class TicketService {
+	/**
+	 * The DAO for the "Ticket" table.
+	 */
+	@Autowired
+	private TicketDao ticketDao;
+	/**
+	 * Logger for handling errors in the DAO layer.
+	 */
+	private static final Logger LOGGER = Logger.getLogger(TicketService.class.getName());
+	/**
+	 * Cancel Ticket function.
+	 */
+	public void cancelReservation(int flight, int row, char seat){
+
+			ticketDao.cancel(flight, row, seat);

```

Comment:
The service layer is the place I would expect the translation from user-provided parameters to persistence-layer-key-type to take place, not in the DAO ...